README.md

```diff
@@ -1,10 +1,92 @@
-# Mailsac Self-Hosted Temporary Email
+# Mailsac Self-Hosted Temporary Email User Interface
 
 This project is a user interface for disposable email. It uses mailsac.com as the backend mail service.
 
+![Checking mail](images/example_animation.gif)
+
+## Use Cases
+
+[Mailsac](https://mailsac.com) already offers disposable email without a need to sign up for an account. What need does
```

Comment:
Needs a header about Mailsac.com limitations or that this starts with “Background” info. I expected to read a useful feature after the header above, but it’s only limitations.